src/utils/assets.js

```diff
@@ -11,7 +11,7 @@
 /**
  * Utility functions for working with Assets
  */
-import { baseUrl } from 'docc-render/utils/theme-settings';
+const { baseUrl } = window;
```

Comment:
I dont think this is necessary?

Reply:
Importing the baseURL from theme-settings creates a dependency cycle between `url-helper`, `theme-settings` and `assets` util files. More info: https://github.com/import-js/eslint-plugin-import/blob/v2.23.4/docs/rules/no-cycle.md 

Since window is a global variable, I decided to just take it as it is.